.github/vale/Vocab/Backstage/accept.txt

```diff
@@ -235,6 +235,7 @@ preconfigured
 prepack
 Preprarer
 productional
+projectId
```

Comment:
Let's get rid of this here and use in-line code blocks wherever it's used in docs.

Reply:
TIL ...